src/Uno.UI/UI/Xaml/Controls/Primitives/ButtonBase/ButtonBase.iOS.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Windows.UI.Xaml.Controls.Primitives
 {
-	public abstract partial class ButtonBase : ContentControl
+	public partial class ButtonBase : ContentControl
```

Comment:
Is there a reason why ButtonBase shouldn't be abstract?

Reply:
@nickrandolph It doesn't appear to be abstract in WinUI: https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.primitives.buttonbase?view=windows-app-sdk-1.2